### PR TITLE
Hooli-style Popup

### PR DESCRIPTION
**Pull request by JuliaNovak**
Hooli-style Popup
[Demo](https://julianovak.github.io/Hooli-style-Popup/) |
[Code base](https://github.com/JuliaNovak/Hooli-style-Popup)

### CODE REVIEW

File: submissions/JuliaNovak/html-css-popup/index.html [hunk 1]
```diff
@@ -0,0 +1,91 @@
+<!DOCTYPE html>
```
Comment (kasionio) [inline submissions/JuliaNovak/html-css-popup/index.html:1]: Please make intendants less for better readability

**Review by kasionio — CHANGES_REQUESTED** [review]
Super minimal design :D 
Can you make it more similar to some menu? Suggestions:
1. Please add padding inside the container
2. Remove padding on hover at the bottom link
3. Remove link styling in labels text
4. `More` button should disappear after clicking on it

File: submissions/JuliaNovak/html-css-popup/index.html (hunk 1 above)
Comment (JuliaNovak) [reply submissions/JuliaNovak/html-css-popup/index.html:1]: what is "intendant"?

Comment (kasionio) [reply submissions/JuliaNovak/html-css-popup/index.html:1]: Sorry, my bad :) Fixed it

Comment (JuliaNovak) [reply submissions/JuliaNovak/html-css-popup/index.html:1]: thanks! also, talking about the 2nd change (Remove padding on hover at the bottom link), the padding appears only while using Tab. If you simply hover over the bottom link using a mouse, there will be no padding. Could you probably give a hint on what's going on there?

Comment (kasionio) [reply submissions/JuliaNovak/html-css-popup/index.html:1]: If I hover with a mouse it also appears. Take a look at CSS properties.
Also related topic - https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity

File: submissions/JuliaNovak/html-css-popup/styles.css [hunk 2]
```diff
@@ -0,0 +1,189 @@
...
+    background-color: lightblue;
+}
+
+.nav-items-container {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+}
+
+.menu li{
```
Comment (kasionio) [inline submissions/JuliaNovak/html-css-popup/styles.css:26]: Avoid styling by HTML tags.
https://frontstuff.io/you-need-to-stop-targeting-tags-in-css

**Review by kasionio — CHANGES_REQUESTED** [review]
Great job! Especially with mobile view. But still need some improvements:
Please take a look at the mobile view, some redundant space at the bottom appears. Also, outline is not centered on chechbox 
image

And make containers equal over popup
image

`More` button is 'jumping' on hover and doesn't have a left border on hover
image

File: submissions/JuliaNovak/html-css-popup/styles.css [hunk 3]
```diff
@@ -0,0 +1,199 @@
+:root {
+    --hover-color: rgb(103, 103, 200);
+    --bg-color:rgb(245, 243, 243);
+    --secondary-bg-color: 	rgb(211,211,211);
```
Comment (kasionio) [inline submissions/JuliaNovak/html-css-popup/styles.css:4]: too many spaces :)

File: submissions/JuliaNovak/html-css-popup/styles.css [hunk 4]
```diff
@@ -0,0 +1,199 @@
...
+a:hover, a:focus, .popup-item img:hover, .popup-item:focus {
+    color: var(--hover-color);
+    outline: 2px solid var(--hover-color);
+    border-radius: 5px;
+}
+
+.popup-item, .popup-label, .input-more {
+    cursor: pointer;
+    position: absolute;
+    /* width: 37px; */
```
Comment (kasionio) [inline submissions/JuliaNovak/html-css-popup/styles.css:58]: you can remove it if it is not used

File: submissions/JuliaNovak/html-css-popup/styles.css [hunk 5]
```diff
@@ -0,0 +1,199 @@
...
+
+.even-more {
+    align-self: center;
+}
+
+@media screen and (max-width: 477px) {
+    .dropdown {
+        right: 3rem;
+    }
+
```
Comment (kasionio) [inline submissions/JuliaNovak/html-css-popup/styles.css:186]: please fix spaces here

File: submissions/JuliaNovak/html-css-popup/styles.css [hunk 6]
```diff
@@ -0,0 +1,199 @@
...
+.item {
+    align-self: baseline;
+    padding: 3px;
+}
+
+.item a{
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+	font-size: 18px;
```
Comment (kasionio) [inline submissions/JuliaNovak/html-css-popup/styles.css:117]: Prettier will help :)

File: submissions/JuliaNovak/html-css-popup/styles.css [hunk 7]
```diff
@@ -0,0 +1,199 @@
...
+    padding: 0;
+}
+
+img {
+    max-width: 3.75rem;
+    max-height: 3rem;
+    margin: 0.55rem;
+}
+
+.item img {
```
Comment (kasionio) [inline submissions/JuliaNovak/html-css-popup/styles.css:103]: it is not a scalable decision. Try to play with images, you can try `fit-object` property for example

File: submissions/JuliaNovak/html-css-popup/styles.css [hunk 8]
```diff
@@ -0,0 +1,199 @@
...
+
+.wrapper{
+    display: grid;
+    grid-template-columns: repeat(3, 1fr);
+    grid-template-rows: repeat(4, 1fr);
+    margin-bottom: 1.5rem;
+    padding: 0;
+}
+
+img {
```
Comment (kasionio) [inline submissions/JuliaNovak/html-css-popup/styles.css:97]: please avoid styling with tags
https://frontstuff.io/you-need-to-stop-targeting-tags-in-css

**Review by kasionio — CHANGES_REQUESTED** [review]
Nice job! Let's make more fixes. We have some weird focus border, take a look
image

And look at the comments bellow

**Review by kasionio — CHANGES_REQUESTED** [review]
Seems you have deleted CSS file accidentally. And readme is redundant here, you can add it into your own repository

File: submissions/JuliaNovak/html-css-popup/index.html [hunk 9]
```diff
@@ -0,0 +1,137 @@
...
+                <li class="menu-item">
+                    <input type="checkbox" id="check" class="popup-item"/>
+                    <label for="check" class="popup-label">
+                        <img src="./img/popup-button.png" alt="popup button"/>
+                    </label>
+                    <div class="dropdown">
+                        <ul class="wrapper">
+                            <li class="item">
+                                <a href="#" class="dropdown-link">
+                                    <img src="./img/calendar.png" alt="calendar" class="dropdown-image">Calendar
```
Comment (kasionio) [inline submissions/JuliaNovak/html-css-popup/index.html:29]: correct formatting 
image

File: submissions/JuliaNovak/html-css-popup/styles.css [hunk 10]
```diff
@@ -0,0 +1,197 @@
...
+.menu-item{
+    display: inline-block;
+    margin-right: 1rem;
+}
+
+.special-item {
+    margin-right: 1.9rem;
+}
+
+.menu-image{
```
Comment (kasionio) [inline submissions/JuliaNovak/html-css-popup/styles.css:35]: this property duplicates in line 98

File: submissions/JuliaNovak/html-css-popup/styles.css [hunk 11]
```diff
@@ -0,0 +1,197 @@
...
+    align-items: center;
+    justify-content: space-between;
+}
+
+.menu-item{
+    display: inline-block;
+    margin-right: 1rem;
+}
+
+.special-item {
```
Comment (kasionio) [inline submissions/JuliaNovak/html-css-popup/styles.css:31]: strange naming :) why special?

**Review by kasionio — CHANGES_REQUESTED** [review]
Still some updates are needed. Let's make images looks not so weird. Please try to use other approaches more scalable, e.g. `object-fit` 
image

**Review by kasionio — APPROVED** [review]
Good job! Let's move forward 🚀

**Comment by kasionio** [comment]
Advice - Try to give commits meaningful names. It is hard but really needed :) In real projects, you will have a bunch of commits and later you will search through them or back to some commits, so you need to understand what commit contains what work. For example, the last one can be named `fit images, fix indentations.` Maybe it's bad naming, but better for sure than "do changes"

**Pull request merged**